### PR TITLE
Check if 'helm-docs' is available

### PR DESCRIPTION
**Pull request by NotMyFault**
### What does this PR do?

The change proposed checks that `helm-docs` is available, and if not, notifies the user to install it:
Screenshot 2024-04-01 at 23 53 13

If you modified files in the `./charts/jenkins/` directory, please also include the following:

```[tasklist]
### Submitter checklist
- [ ] I bumped the "version" key in `./charts/jenkins/Chart.yaml`.
- [ ] I added a new changelog entry to `./charts/jenkins/CHANGELOG.md`.
- [ ] I followed the [technical requirements](https://github.com/jenkinsci/helm-charts/blob/main/CONTRIBUTING.md#technical-requirements).
- [ ] I ran .github/helm-docs.sh from the project root.
```

### CODE REVIEW

**Review by lemeurherve — APPROVED** [review]

File: .github/helm-docs.sh [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 set -euxo pipefail
 
+# check if helm-docs is installed
+if ! command -v helm-docs &> /dev/null
+then
```
Comment (kvanzuijlen) [inline .github/helm-docs.sh:7]:
```suggestion
if ! command -v helm-docs &> /dev/null; then
```

To make the linter happy

File: .github/helm-docs.sh [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+    echo "helm-docs could not be found. Did you install it?"
```
Comment (kvanzuijlen) [inline .github/helm-docs.sh:8]:
```suggestion
    echo "helm-docs could not be found. You can install it from https://github.com/norwoodj/helm-docs"
```

**Review by timja — APPROVED** [review]

File: .github/helm-docs.sh [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 set -euxo pipefail
 
+# check if helm-docs is installed
+if ! command -v helm-docs &> /dev/null; then
+    echo "helm-docs could not be found. You can install it from https://github.com/norwoodj/helm-docs."
+    exit
```
Comment (timja) [inline .github/helm-docs.sh:8]:
```suggestion
  echo "helm-docs could not be found. You can install it from https://github.com/norwoodj/helm-docs."
  exit
```

**Review by kvanzuijlen — APPROVED** [review]
LGTM, just 1 linter issue, but not sure if I personally agree with it.

**Pull request merged**